live/tests/integration/components/feedback-panel-test.js

```diff
@@ -136,15 +147,27 @@ describe('Integration | Component | feedback-panel', function () {
 
     it('clicking on "send" button should save the feedback into the store / API and display the "mercix" view', function () {
       // given
+      const CONTENT_VALUE = 'Prêtes-moi ta plume, pour écrire un mot';
+      const EMAIL_VALUE = 'myemail@gemail.com';
       const $content = this.$('.feedback-panel__field--content');
-      $content.val('Prêtes-moi ta plume, pour écrire un mot');
+      const $email = this.$('.feedback-panel__field--email');
+      $content.val(CONTENT_VALUE);
+      $email.val(EMAIL_VALUE);
       $content.change();
+      $email.change();
 
       // when
       this.$(BUTTON_SEND).click();
+
       // then
       return wait().then(() => {
         expect(isSaveMethodCalled).to.be.true;
+        expect(saveMethodUrl).to.equal('feedback');
+        expect(_.isObject(saveMethodBody)).to.equal(true);
+        expect(saveMethodBody.assessement).to.exists;
```

Comment:
ici, il ne s'agit pas d'un élément jQuery qui renvoie toujours quelque chose donc on peut utiliser `to.exist`.